### ISSUE TITLE
Task project not listed when not present

### ISSUE THREAD

**Issue by stacksjb** [alanvardy/tod#749]
If I am using a filter (such as tod list process -f '@Morning')

And the task is not in the local config, instead of listing the project, it will say "# Project not in config"

Ideally, the project name should still be listed for context - is this because the lookup of project ID to name is happening locally, and the result doesn't exist?

**Comment by alanvardy** [comment]
That's right, currently the lookup is local only so it is only checking the config file for a project with a matching ID.

I could look into doing an async API call to get a list of projects

**Comment by alanvardy** [comment]
Oh that is an interesting idea. What if I have a configuration option to choose one of:

- Prompt to add or remove projects
- Automatically add or remove projects
- Don't check for adding or removing projects (what it is right now)

^ I could occasionally run this check async

I'm one of the people who likes to only have a subset of my projects available locally, but if someones workflow involves creating and deleting a lot I could see this being very useful.

Thoughts?

**Comment by alanvardy** [comment]
For now I will give instuctions to use `tod project import --auto` when this error occurs.

**Comment by alanvardy** [comment]
> I think closing this once the auto project import exists might be ok - especially if there is some sort of default import of all projects.

Sounds good, we can open another issue later if need be. For now the resolution of the problem should be quick and easy.

**Comment by stacksjb** [comment]
Not to overbeat this, but now if I print out several tasks, it prints out once per task
![image](https://github.com/alanvardy/tod/assets/2865491/ac744cf3-6ab0-41c3-8d31-a059c8d02fc5)


How difficult would it be to continue to print "project not found" and then print the line once at the end (basically as an error/notice) instead of on each task?

**Comment by alanvardy** [comment]
Haha, I'll figure out a way to put it only once into the error channel.

https://github.com/alanvardy/tod/issues/763
